docs/devel/build-tool.md

```diff
@@ -104,3 +104,11 @@ by CI.
 
 Starts the build container. This is done automatically by other
 `build/cmd.sh` commands that need the build container.
+
+## e2e
+
+Runs Virtlet e2e tests against the currently running DIND cluster.
+You can also pass test options to this command:
+```
+build/cmd.sh e2e -test.v -ginkgo.focus="Should have default route"
```

Comment:
-ginkgo.focus="Should have default route"
why is this? The same for documentation